source/Options.cpp

```diff
@@ -294,6 +296,13 @@ Options::Options(const boost::program_options::variables_map& variables) {
       : ExportOriginsMode::OnlyOnOrigins;
   propagate_across_arguments_ =
       variables.count("propagate-across-arguments") > 0;
+
+  if (!variables["heuristics"].empty()) {
+    std::string heuristics_path =
+        check_path_exists(variables["heuristics"].as<std::string>()) ;
+    Json::Value json = JsonReader::parse_json_file(heuristics_path);
+    heuristics_ = Heuristics::from_json(json);
```

Comment:
nit: we usually use `auto` in these cases:

```
    auto heuristics_path =
        check_path_exists(variables["heuristics"].as<std::string>());
    auto json = JsonReader::parse_json_file(heuristics_path);
```

Reply:
Solved in 217a2703ab94fc88f24736817225d41ac9dd1b0d